doc/make.jl

```diff
@@ -384,18 +433,7 @@ function Documenter.Writers.HTMLWriter.expand_versions(dir::String, v::Versions)
     cd(() -> filter!(!islink, available_folders), dir)
     filter!(x -> occursin(Base.VERSION_REGEX, x), available_folders)
 
-    # Look for docs for an "active" release candidate and insert it
-    vnums = [VersionNumber(x) for x in available_folders]
-    master_version = maximum(vnums)
-    filter!(x -> x.major == 1 && x.minor == master_version.minor-1, vnums)
-    rc = maximum(vnums)
-    if !isempty(rc.prerelease) && occursin(r"^rc", rc.prerelease[1])
-        src = "v$(rc)"
-        @assert src ∈ available_folders
-        push!(v.versions, src => src, pop!(v.versions))
```

Comment:
So the old code modified `v.versions`, inserting an element just before the last entry (it does `pop!` to remove the last entry, then `push!` to insert `src`, and re-insert the entry that was just opped.

In contrast your new code ignores `v.versions` completely. Perhaps that's OK, I don't know, but it's a difference that I'd suggest to check carefully.